atomic_reactor/util.py

```diff
@@ -852,4 +852,145 @@ def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, par
     return dfparser
 
 
+class Labels(object):
```

Comment:
basically this class should remove need of any dictionaries for new/old labels which are used in whole atomic reactor